File: TypeScript Assignments/3/array.js
var arr;
arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
function arrayOperations(arr) {
    var min = arr[1];
    var max = arr[1];
    var sum = 0;
    for (var _i = 0, arr_1 = arr; _i < arr_1.length; _i++) {
        var num = arr_1[_i];
        if (num > max) {
            max = num;
        }
        if (num < min) {
            min = num;
        }
        sum = sum + parseInt(num);
        console.log(num);
    }
    var length = arr.length;
    var avg = sum / length;
    console.log("min :" + min);
    console.log("max :" + max);
    console.log("avg :" + avg);
}
console.log(arrayOperations(arr));

File: Day 1/Array.java

public class Array {
	
	int[] array1 = {1,5,10,15,20};
	
	public int[] subArr(int start, int end, int[] arr1)
	{
		int size= end -start +1;
		int arr2[]= new int[size];
		for(int i=0;i<size;i++)
			arr2[i]=arr1[start+i];
		return arr2;
	}
	
	public void evenElements(int[] arr1)
	{
		for(int i : arr1)
		{
			if(i%2==0)
				System.out.println(i);
		}
	}
	
	public static void main(String[] args) {
		Array arr= new Array();
		int sum=0;
		
		for(int i : arr.array1)
			sum+=i;
		
		float avg = (float)sum/arr.array1.length;
		
		System.out.println("sum="+sum+"  "+"average="+avg);
		
		int SquareArr[] = new int[5];
		
		System.out.println("Squares :");
		
		for(int i=0; i<arr.array1.length; i++) {
			SquareArr[i] = arr.array1[i]*arr.array1[i];
			System.out.println(SquareArr[i]);
		}
		
		int arr3[]= arr.subArr(1, 4, SquareArr);
		for(int i : arr3)
			System.out.println(i);
		
		arr.evenElements(SquareArr);
}
}
